src/scripts/release.zig

```diff
@@ -429,14 +435,37 @@ fn build_multiversion_body(
 
         try body_file.writeAll(past_binary_contents);
 
+        // The custom release from #1935: https://github.com/tigerbeetle/tigerbeetle/pull/1935.
+        const git_commit: [20]u8 = blk: {
```

Comment:
So we can encode the git commit as binary rather than a hex string inside the header, and save 20 * 63 (1260) bytes.